cmd/lncli/cmd_pay.go

```diff
@@ -403,6 +403,53 @@ func sendPaymentRequest(ctx *cli.Context,
 	}
```

Comment:
nit: as there's multiple calls in this file, maybe better to call it cmd_payments.go ?

Reply:
My plan was to only put the commands in here that initiate a payment (and track payment, which is an extension of send payment). Therefore I chose the verb "pay". There are more payment-related commands like queryroutes, listpayments, etc. I think it is best to stick to one file per command as much as possible.

Reply:
The reason for making an exception to the single command per file rule is that there is code shared.

Reply:
Hmm or should I do more splitting?

Reply:
my personal preference would be to end up with one file per command, but this is already a good step in that direction so either is fine with me

Reply:
yeah not a deal breaker, more like a nit. I'm OK with this totally.